packages/common-scripts/src/locktime_pool.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Please avoid `no-non-null-assertion`, it makes troubleshooting difficult

```ts
function nonNullable<T>(x: T, message = "param cannot be null"): asserts x is NonNullable<T> {
  if (x == null) throw new Error(message)
}
```

Reply:
But in this file it have a lot no-non-null-assertion before I change it🤣